src/dmtcprestartinternal.cpp

```diff
@@ -477,7 +477,7 @@ runMtcpRestart(int fd, RestoreTarget *restoreTarget)
   publishKeyValueMapToMtcpEnvironment(restoreTarget);
```

Comment:
Minor, but maybe a better commit msg is:
> Including riscv in some internal functions

[ Also, note that we usually capitalize the first word of a commit msg, unless it's the name of some code. ]

Reply:
And now that I think about it, a better commit msg might be:
> including riscv in some kernel functions (SYS_*)